admin_guide/limits.adoc

```diff
@@ -31,8 +31,15 @@ default value is applied to the resource.
 [[limit-range-def]]
 // tag::admin_limits_sample_definitions[]
 
-.Core Limit Range Object Definition
+ifdef::openshift-online[]
+[IMPORTANT]
+====
+For {product-title} Pro, the maximum pod memory is 3Gi. The minimum pod or
+container memory that you can specify is 100Mi.
```

Comment:
Lets add similar info for Starter:
Starter Min Pod Memory: 200Mi
Starter Max Pod Memory: 1Gi